src/extractor/CaretValueRuleExtractor.ts

```diff
@@ -199,16 +199,19 @@ export class CaretValueRuleExtractor {
   static processResource(
     input: any,
     fisher: utils.Fishable,
-    resourceType: 'ValueSet' | 'CodeSystem',
-    config: fshtypes.Configuration
+    resourceType: 'ValueSet' | 'CodeSystem' | 'Concept',
+    config?: fshtypes.Configuration
   ): ExportableCaretValueRule[] {
     const caretValueRules: ExportableCaretValueRule[] = [];
     const flatVS = getPathValuePairs(input);
     Object.keys(flatVS)
       .filter(
         key =>
           !RESOURCE_IGNORED_PROPERTIES[resourceType].some(
-            property => key === property || new RegExp(`${property}(\\[\\d+\\])?\\.`).test(key)
+            property =>
+              key === property ||
+              new RegExp(`${property}(\\[\\d+\\])?\\.`).test(key) ||
+              key.startsWith(`${property}[`)
```

Comment:
This seems really similar to the line above, I'm wondering if maybe something like: 
```
new RegExp(`${property}(\\[\\d+\\]|\\.)`).test(key)
```
could capture all of the cases we need.